content/integrations/ecs.md

```diff
@@ -3,6 +3,7 @@ title: Datadog-AWS ECS Integration
 integration_title: AWS ECS
 kind: integration
 git_integration_title: amazon_ecs
+description: "Monitor container statuses, track resource usage, and more."
```

Comment:
AWS ECS integration doesn't monitoring individual containers but rather ECS clusters and tasks.